webapp/src/ts/services/unread-records.service.ts

```diff
@@ -97,10 +104,9 @@ export class UnreadRecordsService implements OnDestroy {
 
   init(callback) {
     this.callback = callback;
-    // wait for db.info to avoid uncaught exceptions: #3754
-    this.dbService
-      .get()
-      .info()
+    // wait for meta db sync to avoid querying meta view before the ddoc was synced from the server
+    this.inited = this.dbSyncService
+      .syncMeta()
```

Comment:
This breaks offline first, right?

I think the correct way to do this is to `get` the ddoc from the local db and if it doesn't exist, return an unknown state and try again next time count is called. Or just query the view and ignore the error...